agr_literature_service/api/tests/test_013_reference_cont.py

```diff
@@ -34,25 +37,100 @@ def test_reference_create_with_existing_items():
         ],
         "resource": 'AGR:AGR-Resource-0000000001',
         "title": "Another title",
-        "volume": "433",
+        "volume": "013a",
         "open_access": True
     }
-    assert 1 == 1
-    # process the reference.
-    reference = ReferenceSchemaPost(**full_xml)
-    res = create(db, reference)
-    assert res == 'AGR:AGR-Reference-0000000005'
+    # process the references
+    ref1 = ReferenceSchemaPost(**full_xml)
+    print(ref1)
+    res1 = create(db, ref1)
```

Comment:
res might be confusing with resource, but it makes sense that it's result, so I think it's fine.  If the test ever did anything with resources besides being part of the reference object, it might be confusing.